mllib/src/main/scala/org/apache/spark/ml/classification/LinearSVC.scala

```diff
@@ -157,7 +157,7 @@ class LinearSVC @Since("2.2.0") (
 
   /**
    * Set block size for stacking input data in matrices.
-   * Default is 4096.
+   * Default is 1024.
```

Comment:
So this will become the new default for LinearSVC too? OK if so.

Reply:
Yes, the new impl of LinearSVC with blocks is quite similar to LR.